### ISSUE TITLE
CurvedNavigationBarState bug

### ISSUE THREAD

**Issue by heshesh2010** [rafalbednarczuk/curved_navigation_bar#70]
Compiler message:
    ../../.pub-cache/hosted/pub.dartlang.org/curved_navigation_bar-0.3.2/lib/curved_navigation_bar.dart:95:19: Error: The getter 'Overflow' isn't defined for the class 'CurvedNavigationBarState'.
     - 'CurvedNavigationBarState' is from 'package:curved_navigation_bar/curved_navigation_bar.dart' ('../../.pub-cache/hosted/pub.dartlang.org/curved_navigation_bar-0.3.2/lib/curved_navigation_bar.dart').
    Try correcting the name to the name of an existing getter, or defining a getter or field named 'Overflow'.
            overflow: Overflow.visible,

**Comment by rafalbednarczuk** [comment]
please show code

**Comment by heshesh2010** [comment]
```
@override
  Widget build(BuildContext context) {
    return WillPopScope(
      onWillPop: () async => false,
      child: Scaffold(
        drawer: DrawerWidget(),
        appBar: AppBar(
          backgroundColor: Theme.of(context).accentColor,
          elevation: 0,
          centerTitle: true,
          title: Text(
            widget.currentTitle ??
                settingsRepo.setting?.appName ??
                S.of(context).home,
            style: Theme.of(context)
                .textTheme
                .headline6
                .merge(TextStyle(letterSpacing: 1.3)),
          ),
        ),
        body: widget.currentPage,
        bottomNavigationBar: CurvedNavigationBar(
          key: navBarGlobalKey,
          backgroundColor: Theme.of(context).highlightColor,
          color: Theme.of(context).accentColor,
          // type: BottomNavigationBarType.fixed,
          //    selectedItemColor: Theme.of(context).hintColor,
          // this will be set when a new tab is tapped
          index: 2,
          items: <Widget>[
            _con.notifications.isNotEmpty
                ? Container()
                : Stack(
                    alignment: Alignment.center,
                    children: <Widget>[
                      new Icon(
                        Icons.notifications,
                        size: 30,
                      ),
                      new Positioned(
                        right: 10,
                        child: new Container(
                          padding: EdgeInsets.all(1),
                          decoration: new BoxDecoration(
                            color: Colors.red,
                            borderRadius: BorderRadius.circular(6),
                          ),
                          constraints: BoxConstraints(
                            minWidth: 12,
                            minHeight: 12,
                          ),
                          child: new Text(
                            '4',
                            style: new TextStyle(
                              color: Colors.white,
                              fontSize: 8,
                            ),
                            textAlign: TextAlign.center,
                          ),
                        ),
                      )
                    ],
                  ),

            //   Icon(Icons.notifications, size: 30),
            Icon(Icons.rate_review, size: 30),
            Icon(Icons.home, size: 30),
            Icon(Icons.note_add, size: 30),
            Icon(Icons.chat, size: 30),
          ],
          onTap: (int i) {
            this._selectTab(i);
          },
        ),
      ),
    );
  }
```

**Comment by rafalbednarczuk** [comment]
please show pubspec.yaml file

**Comment by heshesh2010** [comment]
> name: trip_car_owner
description: order Verndor App by Hesham Elnemr
environment:
  sdk: ">=2.2.2 <3.0.0"
module:
  androidX: true
dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
  cupertino_icons: ^0.1.3
  flutter_swiper: ^1.1.6
  mvc_pattern: ^6.0.0
  global_configuration: ^1.5.0
  intl: '>=0.15.8 <=0.16.1'
  firebase_messaging: ^6.0.16
  google_maps_flutter: ^0.5.27
  html: ^0.14.0+3
  shared_preferences: ^0.5.7+2
  location: ^3.0.2
  dynamic_theme: ^1.0.1
  url_launcher: ^5.4.7
  flare_flutter: ^2.0.3
  charts_flutter: ^0.9.0
  google_map_location_picker: ^3.3.2
  liquid_swipe: ^1.4.2
  image_picker: ^0.6.7
  flutter_staggered_grid_view: ^0.3.0
  font_awesome_flutter: ^8.7.0
  flutter_duration_picker: ^1.0.4
  flutter_plugin_android_lifecycle: ^1.0.6
  http: ^0.12.0+4
  cached_network_image: ^2.0.0+1
  flushbar: ^1.10.4
  liquid_pull_to_refresh: ^2.0.0
  rounded_loading_button: ^1.0.6
  flutter_local_notifications: ^1.4.4
  flare_splash_screen: ^3.0.1
  firebase_dynamic_links: ^0.5.1
  overlay_support: ^1.0.4
  flutter_page_transition: ^0.1.6
  page_transition: ^1.1.5
  auto_animated: ^2.1.0
  device_info: ^0.4.2+4
  flutter_datetime_picker: ^1.3.8
  flutter_rounded_date_picker: 1.0.4
  flutter_date_pickers: ^0.1.0
  curved_navigation_bar: ^0.3.2
  badges: ^1.1.1
  dio: ^3.0.9
  dio_flutter_transformer: ^3.0.2 # latest version
dev_dependencies:
  flutter_test:
    sdk: flutter
    - assets/cfg/
    - assets/img/visa.png
    - assets/img/paypal.png
    - assets/img/paypal2.png
    - assets/img/apple_pay.png
    - assets/img/brazil.png
    - assets/img/canada.png
    - assets/img/china.png
    - assets/img/france.png
    - assets/img/germany.png
    - assets/img/italy.png
    - assets/img/netherlands.png
    - assets/img/spain.png
    - assets/img/united-arab-emirates.png
    - assets/img/united-states-of-america.png
    - assets/img/marker.png
    - assets/img/my_marker.png
    - assets/img/mastercard.png
    - assets/img/visacard.png
    - assets/img/wallet.png
    - assets/img/default.png
    - assets/img/default_food.png
    - assets/img/pay_pickup.png
    - assets/img/load.gif
    - assets/img/logo.png
    - assets/img/rate.png

    - assets/flare/
  fonts:
  - family: Tajawal
    fonts:
      - asset: assets/fonts/Tajawal-ExtraLight.ttf
        weight: 100
      - asset: assets/fonts/Tajawal-ExtraLight.ttf
        weight: 200
      - asset: assets/fonts/Tajawal-Light.ttf
        weight: 300
      - asset: assets/fonts/Tajawal-Regular.ttf
        weight: 400
      - asset: assets/fonts/Tajawal-Medium.ttf
        weight: 500
      - asset: assets/fonts/Tajawal-Medium.ttf
        weight: 600
      - asset: assets/fonts/Tajawal-Bold.ttf
        weight: 700
      - asset: assets/fonts/Tajawal-ExtraBold.ttf
        weight: 800
      - asset: assets/fonts/Tajawal-Black.ttf
        weight: 900

**Comment by Vofchuk** [comment]
Im having the same issu... any fix??

**Comment by rafalbednarczuk** [comment]
fixed in 0.3.5
